Features.md

```diff
@@ -0,0 +1,140 @@
+# Reddit Clone
+
+ACCOUNT
+
+Signup with email:
+- takes an email, password and an unique username 
+- creates a new user in the db and stores the password in a hash format
+- sends out access and refresh token(JWT) as response
+
+Signup with Social(gmail):
+- allows anyone to create a new account using their gmail account
```

Comment:
What data does it get from gmail?

What if a user with that email already exists in your db?

Reply:
But would you still store the google data?

Reply:
Also include the specification in this doc